pkg/vpcmodel/grouping.go

```diff
@@ -234,7 +234,8 @@ func vsiOrSubnetsGroupingBySubnetsOrVsis(groupedConnLines *GroupConnLines,
 		if len(nodesList) == 1 { // a single nif on subnet or subnet on vpc is just added to the result (no grouping)
 			res = append(res, nodesList[0])
 		} else { // a set of network interfaces from the same subnet is grouped by groupedNetworkInterfaces object
-			groupedNodes := groupedConnLines.cacheGrouped.getAndSetEndpointElemFromCache(nodesList)
+			var groupedEndpoints groupedEndpointsElems = nodesList
```

Comment:
```suggestion
			groupedEndpoints := groupedEndpointsElems(nodesList)
```

Reply:
`nodeList` is of type `[]EndpointElem` and not of type `groupedEndpoints`; so the above is not equiv and will not work (unless we add a conversion, in which case we ended with messier code)

Reply:
I suggested a conversion, in the form of `groupedEndpointsElems(nodesList)`